packages/server/api/controllers/reviews.controller.js

```diff
@@ -0,0 +1,38 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getAllReviews = async () => {
+  try {
+    const allReviews = await knex.select().table('reviews');
+
+    if (allReviews.length === 0) {
+      throw new HttpError(`No reviews`, 404);
+    }
+    return allReviews;
+  } catch (error) {
+    return error.message;
```

Comment:
By having this catch here, you remove the point of using HttpError, since it will catch that an unwrap it again. 

You could check if the `error` is a instance of `HttpError` and act based on that.